ChechpointStyles.css

```diff
@@ -0,0 +1,78 @@
+
+body { background-color: silver;
+
+}
+
+
+
+nav { display: block;
+     padding: 40px;
+     background-color: green;
+     
+}
+
+a {color: antiquewhite;
+   font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
Definitely, there is no need for this extra long list of fallback font families. Usually, two or three fonts are enough.